src/software/parameter/config/ai_control.yaml

```diff
@@ -34,74 +30,14 @@ AIControlConfig:
         - "ShootOrChip Play"
     description: >-
         Specifies the ai play that should be in use
-  RefboxConfig:
-    OverrideRefboxPlay:
-      type: "bool"
-      default: false
-      description: >-
-          Overrides the current refbox play with the play
-          specified by currentPlay parameter
-    CurrentRefboxPlay:
-      type: "std::string"
-      default: "Halt"
-      description: >- 
-          Specifies the refbox play that should be in use
-    OverrideRefboxDefendingSide:
-      type: "bool"
-      default: true
-      description: >-
-          Overrides the defending side provided by refbox, 
-          with defendingPositiveSide parameter
-    DefendingPositiveSide:
-      type: "bool"
-      default: true
-      description: >-
-          Positive if selected, Negative if unselected
-    OverrideRefboxFriendlyTeamColor:
-      type: "bool"
-      default: true
-      description: >-
-          Overrides the team color provided by refbox, 
-          with friendlyColorYellow parameter
-    FriendlyColorYellow:
-      type: "bool"
-      default: true
-      description: >-
-          Yellow if selected, Blue if unselected
-    FriendlyGoalieId:
-      type: "int"
-      min: 0
-      max: 16
-      default: 0
-      description: >-
-        The id of the friendly goalie
-    EnemyGoalieId:
-      type: "int"
-      min: 0
-      max: 16
-      default: 0
-      description: >-
-        The id of the enemy goalie
-    IgnoreInvalidCameraData:
-      type: "bool"
-      default: false
-      description: >-
-        Whether or not to ignore invalid camera data. If this value is true, any ball or robot detections that are not
-        within the min and max valid x coordinates will be ignored. If this value is false, all data is collected as
-        normal and not ignored.
-    MinValidX:
-      type: "double"
-      min: -10.0
-      max: 10.0
-      default: -10.0
-      description: >-
-        When ignoreInvalidCameraData is true, any robot or ball detection with an x-coordinate less than this value
-        is ignored.
-    MaxValidX:
-      type: "double"
-      min: -10.0
-      max: 10.0
-      default: 10.0
-      description: >-
-        When ignoreInvalidCameraData is true, any robot or ball detection with an x-coordinate greater than this
-        value is ignored.
+  OverrideRefboxPlay:
+    type: "bool"
+    default: false
+    description: >-
+        Overrides the current refbox play with the play
+        specified by currentPlay parameter
```

Comment:
What does `currentPlay` mean? Is it in this `ai_control.yaml` file? Or is it referring to `CurrentRefboxPlay` below?

Reply:
It's the `currentAIPlay`. Good catch, clarified